### ISSUE TITLE
Reconfigure the project in order to rid of Resources bundle

### ISSUE THREAD

**Issue by YuriyVelichkoPI** [prebid/prebid-mobile-ios#795]
Looks like the most optimal way is to get rid of the resource bundle. We see that other SDKs, such as GMA SDK and MAX, do not add resource bundles to the app. 

The following table describes how we can migrate to a resourceless product.

|Resource Name|Migration action|
|----|----|
|`companion.html`| Use a static markup string in the code.|
|`ClickthroughBrowserView.nib`| [Migrate to SFSafariViewController](https://github.com/prebid/prebid-mobile-ios/issues/793), or create UI layout programmaticaly| 
|`Assets.car`|Use base64 encoded data instead of assets|
|`mraid.js`|Download from cloud or CDN on demand|
|`omsdk.js`|Download from cloud or CDN on demand|

_Originally posted by @YuriyVelichkoPI in https://github.com/prebid/prebid-mobile-ios/issues/772#issuecomment-1431155434_

**Comment by YuriyVelichkoPI** [comment]
Hi @bretg, @ChrisHuie, @mmullin

In the scope of this ticket, we would like to host two files on Prebid's S3:
- mraid.js
- omsdk.js

It is a common practice and expected behavior for SDKs to host these files on the web and download them on SDK initialization. 

Can you help us with this?

**Comment by bretg** [comment]
@YuriyVelichkoPI - let's talk about this. AWS cloudwatch is incredibly expensive, and I'm not sure that Prebid wants to have the data trail about SDK usage. 

I would prefer to provide instructions for app developers to host this on their own, providing a config/method to tell the SDK where the files are located.

**Comment by YuriyVelichkoPI** [comment]
@bretg, I see. In this case, let's just keep the content of files in  _string_ objects in the codebase. If we face the necessity of online updates, we introduce this feature in the manner that you proposed. But so far, it is unnecessary and will make SDK configuration more complex from the publisher's perspective.

**Comment by bretg** [comment]
An alternate option would be to host these files on jsdelivr instead of AWS. This is free to us.

e.g. several other Prebid files are stored there:
- https://cdn.jsdelivr.net/npm/prebid.js@latest/dist/not-for-prod/prebid.js
- https://cdn.jsdelivr.net/npm/prebid-universal-creative@latest/dist/creative.js
- https://cdn.jsdelivr.net/gh/prebid/currency-file@1/latest.json

**Comment by YuriyVelichkoPI** [comment]
It totally works for the SDK.

**Comment by bretg** [comment]
Cool - you'll have to figure out how to get the files there... there's not a standard way. The currency file is done in a pretty simple way -- jsdelivr can source it right from github if you'd like.

**Comment by chwo** [comment]
Hello 👋 

Do you have any estimated time of arrival for the version 2.1.1 with this fix?

Because currently we are unable to update the PrebidMobile sdk to version 2.x and are stuck on version 1.12.2:
- SPM integration isn't possible because of https://github.com/prebid/prebid-mobile-ios/issues/640
- CocoaPods integration isn't possible because of this issue https://github.com/prebid/prebid-mobile-ios/issues/772

**Comment by chwo** [comment]
I probably _could_ but I think it is very unusual that the consumer of the the framework has to sign it. However I noticed that the bundle resources issue does no longer occur in the latest CocoaPods version (see https://github.com/CocoaPods/CocoaPods/issues/11402). So switching to the latest CocoaPods version fixed it for me.

**Comment by YuriyVelichkoPI** [comment]
Yep, sure, it's not the best practice, but at least it frees from the blocker.

It's excellent that CocoaPods fix the issue. 

In any case, we will remove the resource bundle next release. As usual, we release new versions once per two months if nothing urgent appears.
